Extensions/audio_plus.js

```diff
@@ -74,13 +69,38 @@ XKit.extensions.audio_plus = {
 					}
 				}
 			};
-			var observer_dock = new MutationObserver(callback);
-			observer_dock.observe(targetNode, config);
+			this.observer_dock = new MutationObserver(callback);
```

Comment:
should be defined on the extension object so the `.disconnect()` never references an undefined key (breaks extension disabling/uninstalling in some cases)